docs/book/mlops-stacks/data-validators/data-validators.md

```diff
@@ -1,94 +1,86 @@
+results. A lot of effort goes into ensuring and maintaining data quality not
+only in the initial stages of model development, but throughout the entire
+machine learning project lifecycle. Data Validators are a category of ML
+libraries, tools and frameworks that grant a wide range of features and best
+practices that should be employed in the ML pipelines to keep data quality in
+check and to monitor model performance to keep it from degrading over time.
+
+Data profiling, data integrity testing, data and model drift detection
+are all ways of employing data validation techniques at different points in your
+ML pipelines where data is concerned: data ingestion, model training and
+evaluation and online or batch inference. Data profiles and model performance
+evaluation results can be visualized and analyzed to detect problems and take
+preventive or correcting actions.
+
+Related concepts:
+
+* the Data Validator is an optional type of Stack Component that needs to be
+registered as part of your ZenML [Stack](../../developer-guide/stacks-profiles-repositories/stack.md).
+* Data Validators used in ZenML pipelines usually generate data profiles and
+data quality check reports that are versioned and stored in the [Artifact Store](../artifact-stores/artifact-stores.md).
+They can be retrieved and inspected using [the post-execution workflow API](../../developer-guide/steps-pipelines/inspecting-pipeline-runs.md).
+
+## When to use it
+
+[Data-centric AI practices](https://blog.zenml.io/data-centric-mlops/) are
+quickly becoming mainstream and using Data Validators are an easy way to
+incorporate them into your workflow. These are some common cases where you
+may consider employing the use of Data Validators in your pipelines:
+
+* early on, even if it's just to keep a log of the quality state of your
+data and the performance of your models at different stages of development.
+* if you have pipelines that regularly ingest new data, you should use data
+validation to run regular data integrity checks to signal problems before
+they are propagated downstream.
+* in continuous training pipelines, you should use data validation techniques to
+compare new training data against a data reference and to compare the
+performance of newly trained models against previous ones.
+* when you have pipelines that automate batch inference or if you regularly
+collect data used as input in online inference, you should use data validation
+to run data drift analyses and detect training-serving skew, data drift and
+model drift.
+
+### Data Validator Flavors
+
+Data Validator are optional stack components provided by integrations. The
+following table lists the currently available Data Validators and summarizes
+their features and the data types and model types that they can be used with in
+ZenML pipelines:
+
+| Data Validator | Flavor | Integration | Validation Features | Data Types | Model Types | Notes |
```

Comment:
This doesn't fit fully on the page, as such the flavor, integration and data validator columns show almost all the same information, might make sense to reorder so that the most important columns are the first 3-4

Reply:
Good point ! I reshuffled it a bit. Should be better.